### PR TITLE
Add PHP support for AAS linux

### PR DESCRIPTION
**Pull request by jcstorms1**
- updates the wrapper script with PHP specific setup

### CODE REVIEW

File: linux/datadog_wrapper [hunk 1]
```diff
@@ -132,6 +135,25 @@ setUpDotnetEnv() {
...
+
+    if [ -z "$DD_PHP_TRACER_VERSION" ]; then
+        DD_PHP_TRACER_VERSION=0.83.1
+    fi
+
+    DD_PHP_TRACER_URL=https://github.com/DataDog/dd-trace-php/releases/download/${DD_PHP_TRACER_VERSION}/datadog-setup.php
+
+    echo "Installing PHP tracer from $DD_PHP_TRACER_URL"
+    if curl -LO --fail "$DD_PHP_TRACER_URL"; then
+        eval "php datadog-setup.php --php-bin=all"
```
Comment (purple4reina) [inline linux/datadog_wrapper:150]: Is the `eval` really necessary? Honest question.

File: linux/datadog_wrapper [hunk 2]
```diff
@@ -90,7 +92,8 @@ setUpBinary() {
     if [ "$BINARY" = "dogstatsd" ]; then BINARY="dogstatsd start"; fi
 
     echo "Starting $BINARY"
-    ./"$BINARY" &
+    eval "${DD_DIR}/${BINARY} &"
```
Comment (purple4reina) [inline linux/datadog_wrapper:95]: This change is confusing to me. How was it working before if it now needs the `$DD_DIR` path? Have you tested this with all the runtimes?

**Review by purple4reina — APPROVED** [review]

File: linux/datadog_wrapper (hunk 2 above)
Comment (jcstorms1) [reply linux/datadog_wrapper:95]: I switched to the absolute path while debugging an issue. I figured it was better to eliminate relative paths moving forward.

File: linux/datadog_wrapper (hunk 1 above)
Comment (jcstorms1) [reply linux/datadog_wrapper:150]: Yes. I was running into issues where the string was not being executed as a command correctly. Using eval allowed the command to execute as intended.

**Pull request merged**